deploy/azure/infra/main.tf

```diff
@@ -1,14 +1,154 @@
-# # Naming convention
-# module "default_label" {
-#   source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=0.24.1"
-#   namespace  = format("%s-%s", var.name_company, var.name_project)
-#   stage      = var.stage
-#   name       = "${lookup(var.location_name_map, var.resource_group_location, "northeurope")}-${var.name_component}"
-#   attributes = var.attributes
-#   delimiter  = "-"
-#   tags       = var.tags
-# }
-
-# module "data" {
-#   source = "git::https://github.com/amido/stacks-terraform//azurerm/modules/azurerm-data-platform?ref=vXXXXXX"
-# }
+
+# Naming convention
+module "default_label" {
+  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=0.24.1"
+  namespace  = format("%s-%s", var.name_company, var.name_project)
+  stage      = var.stage
+  name       = "${lookup(var.location_name_map, var.resource_group_location, "northeurope")}-${var.name_component}"
+  attributes = var.attributes
+  delimiter  = "-"
+  tags       = var.tags
+}
+
+
+resource "azurerm_resource_group" "default" {
+  name     = module.default_label.id
+  location = var.resource_group_location
+  tags     = var.tags
+}
+
+# KV for ADF
+module "kv_default" {
+  source                    = "git::https://github.com/amido/stacks-terraform//azurerm/modules/azurerm-kv?ref=feature/module-kv"
+  resource_namer            = substr(replace(module.default_label.id, "-", ""), 0, 24)
+  resource_group_name       = azurerm_resource_group.default.name
+  resource_group_location   = azurerm_resource_group.default.location
+  create_kv_networkacl      = false
+  enable_rbac_authorization = false
+
+}
+
+# module call for ADF
+module "adf" {
+  source                  = "git::https://github.com/amido/stacks-terraform//azurerm/modules/azurerm-adf?ref=master"
+  resource_namer          = module.default_label.id
+  resource_group_name     = azurerm_resource_group.default.name
+  resource_group_location = azurerm_resource_group.default.location
+  git_integration         = var.git_integration
+}
+
+
+resource "azurerm_role_assignment" "kv_role" {
+  scope                = module.kv_default.id
+  role_definition_name = var.kv_role_adf
+  principal_id         = module.adf.adf_managed_identity
+}
+
+resource "azurerm_role_assignment" "storage_role" {
+  scope                = module.adls_default.storage_account_ids[0]
+  role_definition_name = var.adls_datalake_role_adf
+  principal_id         = module.adf.adf_managed_identity
+}
+
+resource "azurerm_role_assignment" "storage_role_config" {
+  scope                = module.adls_default.storage_account_ids[1]
+  role_definition_name = var.blob_dataconfig_role_adf
+  principal_id         = module.adf.adf_managed_identity
+}
+resource "azurerm_log_analytics_workspace" "la" {
+  name                = var.la_name
```

Comment:
Should this be using the same naming conventions as the other resources

Reply:
have changed this now